client/cypress/integration/query/parameter_spec.js

```diff
@@ -1,3 +1,15 @@
+function openAndSearchAntdDropdown(testId, paramOption) {
```

Comment:
Your function now also selects a value, so, you have two options:
1. Keep the selection separated 
2. If you want to have a function that also selects a value after a given filtering, we should put it into the existing `cy.selectAntdOption()`

I'd rather go with 1 as there are a few AntdOption helpers that we haven't ported yet (`selectAntdOption`, `selectAntdOptionLabel` and `selectAntdOptionByTestId`).

Also note that we don't need to validate that the option exists as `cy.click` already does that.